src/Microsoft.AspNet.Razor/Generator/Compiler/CodeTree/Chunks/TagHelpers/TagHelperChunk.cs

```diff
@@ -12,6 +12,29 @@ namespace Microsoft.AspNet.Razor.Generator.Compiler
     public class TagHelperChunk : ChunkBlock
     {
         /// <summary>
+        /// Instantiates a new <see cref="TagHelperChunk"/>.
+        /// </summary>
+        /// <param name="tagName">The tag name associated with the tag helpers HTML element.</param>
+        /// <param name="attributes">The attributes associated with the tag helpers HTML element.</param>
+        /// <param name="selfClosing">
+        /// The <c>bool</c> indicating whether or not the tag of the tag helpers HTML element is self-closing.
+        /// </param>
+        /// <param name="descriptors">
+        /// The <see cref="TagHelperDescriptor"/>s associated with this tag helpers HTML element.
+        /// </param>
+        public TagHelperChunk(
+            string tagName,
+            IDictionary<string, Chunk> attributes,
+            bool selfClosing,
```

Comment:
nit: it's less important here but `selfClosing` should be before `attributes`. keep `tagName` and `selfClosing` together (and in that order) everywhere we can